server/src/migrations/20210719114220_addCustomerIdCascadeOnDelete.ts

```diff
@@ -0,0 +1,25 @@
+import * as Knex from "knex";
```

Comment:
I'd guess you don't have prettier setup to format on save (it would change to single quotes)

It might be good to do, to avoid unnecessary formatting noise later

Reply:
Ah, didn't know about the format on save, I set it up now, thanks!